src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -130,6 +132,13 @@ function parseEndpoints(
         channel: "string",
       });
       triggered = { eventTrigger: ep.eventTrigger };
+      if (typeof triggered.eventTrigger.channel !== "undefined") {
+        triggered.eventTrigger.channel = resolveChannelName(
+          project,
+          triggered.eventTrigger.channel,
+          defaultRegion
+        );
+      }
```

Comment:
*nit - we could use `proto.renameIfPresent` with `converter` option here instead, but I think that only saves us like 5 key strokes 🤷🏼‍♂️

Reply:
Done.